Documentation/advanced-configuration.md

```diff
@@ -429,6 +430,62 @@ other OSDs holding replica data are unavailable:
 ceph osd primary-affinity osd.0 0
 ```
 
+## OSD Dedicated Network
+
+It is possible to configure ceph to leverage a dedicated network for the OSDs to communicate across. A useful ovewview is the [CEPH Networks](http://docs.ceph.com/docs/master/rados/configuration/network-config-ref/#ceph-networks) section of the Ceph documentation. If you declare a cluster network, OSDs will route heartbeat, object replication and recovery traffic over the cluster network. This may improve performance compared to using a single network.
+
+Two changes are neessary to the configuration to enable this capability:
+
+### Use hostNetwork in the rook ceph cluster configuration
+
+Enable the `hostNetwork` setting in the [Ceph Cluster CRD configuration](https://rook.io/docs/rook/master/ceph-cluster-crd.html).  For example,
+
+```bash
+apiVersion: ceph.rook.io/v1beta1
+kind: Cluster
+metadata:
+  name: rook-ceph
+  namespace: rook-ceph
+spec:
+  dataDirHostPath: /var/lib/rook
+  serviceAccount: rook-ceph-cluster
+  network:
+    hostNetwork: true
+  dashboard:
+    enabled: true
+  # cluster level storage configuration and selection
+  storage:
+    useAllNodes: true
+    useAllDevices: true
+    deviceFilter:
+    location:
+    config:
+      metadataDevice:
+      databaseSizeMB: "1024" # this value can be removed for environments with normal sized disks (100 GB or larger)
+      journalSizeMB: "1024"  # this value can be removed for environments with normal sized disks (20 GB or larger)
+```
+
+### Define the subnets to use for public and private OSD networks
+
+Edit the `rook-config-override` configmap to define the custom network configuration:
+
+```bash
+kubectl -n rook-ceph edit configmap rook-config-override
+```
+
+In the editor, add a custom configuration to instruct ceph which subnet is the piublic network and which subnet is the private network. For example:
+
+```bash
+apiVersion: v1
+data:
+  config: |
+    [global]
+    public network =  10.0.7.0/24
```

Comment:
don't you only want to override the cluster network? the public network could remain as set by the public ip env var set on the pod.